src/framework/components/screen/component.js

```diff
@@ -139,24 +139,19 @@ class ScreenComponent extends Component {
     }
 
     _bindElement(element) {
-        this._elements.push(element);
+        this._elements.add(element);
     }
 
     _unbindElement(element) {
-        const index = this._elements.indexOf(element);
-        if (index !== -1) {
-            this._elements.splice(index, 1);
-        }
+        this._elements.delete(element);
     }
 
     onRemove() {
         this.system.app.graphicsDevice.off("resizecanvas", this._onResize, this);
         this.fire('remove');
 
-        for (let i = 0; i < this._elements.length; i++) {
-            this._elements[i]._onScreenRemove();
-        }
-        this._elements.length = 0;
+        this._elements.forEach(element => element._onScreenRemove());
```

Comment:
Before we use non ES5 feature, we need to ensure it is well compiled to ES5 by Babel, and then added to [CONTRIBUTING.md](https://github.com/playcanvas/engine/blob/master/.github/CONTRIBUTING.md).

Reply:
Arrow functions are converted properly by Babel and already used in other parts of the engine in the conversion to modern JavaScript.

Reply:
I'll add an issue to add that to CONTRIBUTING.md